README.md

```diff
@@ -104,3 +104,44 @@ of rdiff-backup run with the very verbose option `-v9`.
 
 The FAQ in particular is an important reference, especially if you are
 using smbfs/CIFS, Windows, or have compiled by hand on Mac OS X.
+
+
+## CONTRIBUTING
+
+Rdiff-backup is an open source software developed by many people over a long period of time. There is no particular company backing the development of rdiff-backup, so we rely very much on individual contributors who "scratch their itch". **All contributions are welcome!**
+
+There are many ways to contribute:
+
+- Testing, troubleshooting and writing good bug reports that are easy for other developers to read and act upon
+- Reviewing and triaging [existing bug reports and issues](https://github.com/rdiff-backup/rdiff-backup/issues), helping other developers focus their efforts
+- Writing documentation (e.g. the [man page](https://github.com/rdiff-backup/rdiff-backup/blob/master/rdiff-backup.1)), or updating the webpage rdiff-backup.net
+- Packaging and shipping rdiff-backup in your own favorite Linux distribution or operating system
+- Running tests on your favorite platforms and fixing failing tests
+- Writing new tests to get test coverage up
+- Fixing bug in existing features or adding new features
+
+If you don't have anything particular in your mind but want to help out, just browse the list of issues. Both coding and non-coding tasks have been filed as issues.
+
+## Installing the latest development version
+
+When testing and giving feedback, please make sure you are running the latest development version.
+
+### Development version from PyPi
+
+For most platforms the easiest way to install the [latest development version of rdiff-backup is via the Python package installer](https://pypi.org/project/rdiff-backup/#history):
+
+  pip3 install --pre rdiff-backup
+
+Note the `--pre` switch. If it fails to compile, you might also need to install `librsync-dev` first.
+
+### Development version from PPA (for Ubuntu)
```

Comment:
Perhaps add the copr repo from @frankcrawford ? https://copr.fedorainfracloud.org/coprs/frankcrawford/rdiff-backup

Reply:
I did not change the contents here, just formatting, so I will not update the contents now either because then I would need to fully understand/test the docs, which I don't have time for right now (as it would require me to setup Fedora etc). Feel free to update the contents of this section in a future commit.

Reply:
Sure, I'll add